code/game/sound.dm

```diff
@@ -69,7 +86,8 @@
 
 		distance *= distance_multiplier
 
-		S.volume -= max(distance - world.view, 0) * 2 //multiplicative falloff to add on top of natural audio falloff.
+		if(max_distance) //If theres no max_distance we're not a 3D sound, so no falloff.
+			S.volume -= (max(distance - falloff_distance, 0) ** falloff_exponent) / ((max_distance - falloff_distance) ** falloff_exponent) * S.volume
```

Comment:
max distance should be set to the max between it and distance, otherwise **very** dumb shit will happen